src/common/luxon.neo4j.ts

```diff
@@ -31,6 +31,7 @@ DateTime.prototype.toNeo4JDateTime = function(this: DateTime) {
   );
 };
 
+// @ts-ignore FIXME: unclear how to get around the TS error here
```

Comment:
```
Element implicitly has an 'any' type because expression of type 'unique symbol' can't be used to index type 'DateTime'.
  Property '["util".inspect.custom]' does not exist on type 'DateTime'.
```

Reply:
Ok TS is basically saying `DateTime` doesn't have that property. This is JS magic and doesn't change the external API, so I think it's fine to `as any` here.
```ts
(DateTime.prototype as any)[inspect.custom] = function() {};
```

Reply:
Great! Updated and pushed.